Musico/db/migrate/20230322091155_create_group_sessions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateGroupSessions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :group_sessions do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
may be we call it group_admin_id, and point it to user table

```suggestion
      t.references :group_admin, null: false, foreign_key: { to_table: :users }
```

Reply:
Also make suitable changes in the model as well